### PR TITLE
Upgrade vscode-azuretools dependencies

### PR DESCRIPTION
**Pull request by bwateratmsft**
Should help with #2833 / #2843

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -36,49 +35,6 @@ let config = dev.getDefaultWebpackConfig({
     },
 
     loaderRules: [
-        {
```
Comment (karolz-ms) [inline webpack.config.js:39]: OMG, love it, no more webpack goo!! ☺️

Comment (bwateratmsft) [reply webpack.config.js:39]: Gonna have to put some goo back; it's not copying resources from AzExt...

**Review by karolz-ms — APPROVED** [review]
Node package system continues to amaze me: 16K additions and 22K deletions from `package-lock.json` 😮

**Comment by bwateratmsft** [comment]
Closing this, there are too many issues to iron out today and CTI needs a build today. #2833 / #2843 will just have to exist in 1.12.0.

**Comment by bwateratmsft** [comment]
Upstream issue: https://github.com/microsoft/vscode-azuretools/issues/879

**Pull request closed without merging**